transit_odp/templates/pages/accessibility.html

```diff
@@ -265,34 +282,39 @@ <h2 class="govuk-heading-m">{% filter force_escape %}{% trans 'Preparation of th
                         {% filter force_escape %}
                             {% blocktrans %}
                                 We used this approach to deciding on a
-                                sample of pages to test: analysis of the
+                                sample of pages to test the analysis of the
                                 key user journeys on the platform.
```

Comment:
Design says..
'We used this approach to **decide** on a sample'.

Reply:
Fixed this.